### ISSUE TITLE
Preserve query string

### ISSUE THREAD

**Issue by bdrichards** [openseadragon/bookmark-url#1]
This plugin is wonderful! I will use it to allow students to search for items in images and send the URL to me for grading!
However, my URL includes a query string with a parameter that identifies which GigaPan image to load, for example: 
https://br1.nic.edu/giga/gpan/?ID=88438&selinfo="Zoom and center your selection"
gets replaced with
https://br1.nic.edu/giga/gpan/#zoom=2&x=0.5&y=0.051495915756323195

and the ID  and other parameter(s) for the image is lost. Can the hash be appended after the query string and any associated parameters?

EDIT:
I added: 
`searchStr = window.location.search;`
to the parseHash fcn, and then changed the following line in the updateUrl fcn:
`var url = location.pathname + _searchStr_ + '#zoom=' + zoom + '&x=' + pan.x + '&y=' + pan.y;`

This achieves what I need, for now......

**Comment by iangilman** [comment]
Yep, that's exactly the correct solution! Would you mind making a pull request with it, so others can have your fix? :)

**Comment by bdrichards** [comment]
Thanks! I would love to make a pull request, but I have to confess much ignorance about how that works.....

**Comment by iangilman** [comment]
No worries... I'm happy to help walk you through it! Here's our cheat sheet for making pull requests for the OpenSeadragon repository:

https://github.com/openseadragon/openseadragon/wiki/Making-a-Pull-Request

This is a different repository, but otherwise the process is pretty much the same.

Are you up for giving it a try?

**Comment by bdrichards** [comment]
Hello @iangilman !
I certainly appreciate your attempt to help me along with getting the changes I need into the plugin. Currently I just edit the JS code each time there is an update. I did try to follow your suggestions for making a "pull request" correctly and thought I had done so back in 2018
![tempgithub](https://user-images.githubusercontent.com/18133188/168453361-e04aa39b-5685-44f0-b344-5ca1ab6577f1.jpg)

I finally just made changes on a "branch" in github and then submitted a pull request, which seems to still be open? Now that version 0.0.4 is available and I have made the needed changes by hand in our copy of the JS code a fourth time, I thought to try to achieve what you needed again......
Any guidance greatly appreciated. Keep in mind, I have nothing stored locally on my desktop - do I need something local? All the edits I need implemented are in my "fork" on github....
Bill
